addons/main/functions/UnitAction/fnc_doCheckBody.sqf

```diff
@@ -37,14 +37,34 @@ _unit setVariable [QEGVAR(danger,forceMove), true];
     {
         // condition
         params ["_unit", "_body"];
-        (_unit distance _body < 0.7) || {!(_unit call FUNC(isAlive))}
+        (_unit distance _body < 0.8) || {!(_unit call FUNC(isAlive))}
     },
     {
         // on near body
         params ["_unit", "_body"];
         if (_unit call FUNC(isAlive)) then {
             [QGVAR(OnCheckBody), [_unit, group _unit, _body]] call FUNC(eventCallback);
             _unit action ["rearm", _body];
+
+            // get backpack
+            if ((backpack _unit) isEqualTo "" && {backpack _body isNotEqualTo ""}) then {
+                private _items = backpackItems _body;
+                private _backpack = backpack _body;
+                removeBackpack _body;
+                _unit addBackpack _backpack;
```

Comment:
Hmm, shouldn't these be the global variants ?

^ [The latter](https://community.bistudio.com/wiki/addBackpackGlobal)'s docs warn you need to `waitUntil` cuz the backpack will be null for a bit on a non-local unit, though, but I don't see why that would be a problem.
```suggestion
                removeBackpackGlobal _body;
                _unit addBackpackGlobal _backpack;
```

Reply:
Nice change. Because we will only be adding backpacks to LOCAL units, we don't need to worry about waiting.